packages/data-point/lib/reducer-types/reducer-helpers/utils/index.js

```diff
@@ -0,0 +1,44 @@
+const last = require('lodash/last')
+
+/**
+ * used by ReducerDefault to check
+ * when the default should be used
+ * @param {*} value
+ * @return {boolean}
+ */
+function reducerOutputIsFalsy (value) {
+  return (
+    value === null ||
+    typeof value === 'undefined' ||
+    Number.isNaN(value) ||
+    value === ''
+  )
+}
+
+module.exports.reducerOutputIsFalsy = reducerOutputIsFalsy
+
+/**
+ * used by ReducerFilter and ReducerFind
```

Comment:
more than 'used by' (this could get diff to maintain, as others start using it ... or not using it. Either remove the description or talk about what the code does... 

which I feel it makes more sense to have the name of the function describe itself ( which already does) 
and add some comments specially on the second return on what they do to add more clarity.